.circleci/config.yml

```diff
@@ -277,7 +277,7 @@ commands:
                   COMMIT_SHA=$(git rev-parse --short HEAD)
                   LAST_RELEASE=$(git describe --tags --abbrev=0)
                   if [[ -n ${CIRCLE_BRANCH+x} ]] && [[ $CIRCLE_BRANCH != "main" ]] && [[ "$(git log -1)" = *"publish_android_snapshot"* ]]; then
-                    sed -i -e "s/^VERSION_NAME=.*/VERSION_NAME=${LAST_RELEASE:9}-${CIRCLE_BRANCH}-SNAPSHOT/" gradle.properties
+                    sed -i -e "s/^VERSION_NAME=.*/VERSION_NAME=${LAST_RELEASE:9}-${CIRCLE_BRANCH}-${COMMIT_SHA}-SNAPSHOT/" gradle.properties
```

Comment:
guess we need to make sure it's short commit sha and not the long one

Reply:
`COMMIT_SHA=$(git rev-parse --short HEAD)` it is, line 276